substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -279,18 +288,55 @@ pub struct VerifyCmd {
 	pub input_chain_spec: PathBuf,
 }
 
-/// Processes `CreateCmd` and returns JSON version of `ChainSpec`.
-pub fn generate_chain_spec_for_runtime(cmd: &CreateCmd) -> Result<String, String> {
-	let code = fs::read(cmd.runtime_wasm_path.as_path())
-		.map_err(|e| format!("wasm blob shall be readable {e}"))?;
+#[derive(Deserialize, Serialize, Clone)]
+pub struct ParachainExtension {
+	/// The relay chain of the Parachain.
+	pub relay_chain: String,
+	/// The id of the Parachain.
+	pub para_id: u32,
+}
 
-	let chain_type = &cmd.chain_type;
+/// Creates a new spec for a parachain.
+fn create_para_chain_spec(
+	cmd: &CreateCmd,
+	code: &[u8],
+	chain_type: ChainType,
+) -> sc_chain_spec::ChainSpecBuilder<ParachainExtension> {
+	let relay_chain = &cmd.relay_chain.clone().expect("No relay chain provided");
+	let para_id = *&cmd.para_id.expect("No Para ID provided");
 
-	let builder = GenericChainSpec::<()>::builder(&code[..], Default::default())
+	let mut properties = sc_chain_spec::Properties::new();
+	properties.insert("tokenSymbol".into(), "UNIT".into());
+	properties.insert("tokenDecimals".into(), 12.into());
+	properties.insert("ss58Format".into(), 42.into());
```

Comment:
I am pretty sure this is not used by anyone, and is now part of `frame_system::Config`.

Reply:
I still see chain specs using this, so maybe this is a recent thing - I tried looking but could not find it, any chance you could share a link?

Reply:
I tried looking for these parameters in `frame_system::Config` but could not find them.  I guess if we do include them, it would be appropriate to offer flags for them in the CLI anyway, but I can remove them for now

Reply:
https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/trait.Config.html#associatedtype.SS58Prefix

Reply:
removed